client/src/Clusters/Footer/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import '../styles.css';
+import './Footer.style.css';
+import {data} from './mock_data.js';
```

Comment:
I think we can call the mock_data `data.js` instead since we won't be using a backend for it